packages/Input/index.vue

```diff
@@ -19,7 +19,7 @@
           ]"
           :placeholder="placeholder"
           @change="handleChange"
-          @keyup="handleKeyup"
+          @input="handleInput"
```

Comment:
What exactly is this issue 🧐

Reply:
Open [this link](https://e5885.csb.app/) with a mobile phone, and try to type some Chinese words.